app/services/sign_in/constants/auth.rb

```diff
@@ -26,7 +26,10 @@ module Auth
       CLIENT_STATE_MINIMUM_LENGTH = 22
       CODE_CHALLENGE_METHOD = 'S256'
       CSP_TYPES = [IDME = 'idme', LOGINGOV = 'logingov', DSLOGON = 'dslogon', MHV = 'mhv'].freeze
-      OPERATION_TYPES = [SIGN_UP = 'sign_up', AUTHORIZE = 'authorize'].freeze
+      OPERATION_TYPES = [SIGN_UP = 'sign_up',
```

Comment:
When we copy `OPERATION_TYPES` to the sessions controller, we can leave out `sign up` since that is apparently deprecated (we eventually want to remove it from these constants too but let's not worry about it now here)